src/portal/src/app/base/project/repository/artifact/artifact-additions/artifact-vulnerabilities/artifact-vulnerabilities.component.html

```diff
@@ -11,14 +11,16 @@
             <clr-dg-action-bar>
                 <div class="clr-row center">
                     <div class="ml-05">
-                        <button id="scan-btn" (click)="scanNow()" type="button"  class="btn  btn-secondary" [clrLoading]="scanBtnState" [disabled]="!(hasEnabledScanner && hasScanningPermission && !onSendingScanCommand)">
-                            <clr-icon shape="shield-check" size="16"></clr-icon>&nbsp;
-                            <span>{{'VULNERABILITY.SCAN_NOW' | translate}}</span>
+                        <button id="scan-btn" (click)="scanOrStop()" type="button"  class="btn  btn-secondary" [clrLoading]="scanBtnState" [disabled]="!(hasEnabledScanner && hasScanningPermission && !onSendingScanCommand)">
```

Comment:
a method can be defined to represent the button status conditions: `!(hasEnabledScanner && hasScanningPermission && !onSendingScanCommand)`

Reply:
updated